docs/serving/samples/hello-world/helloworld-java-spring/README.md

```diff
@@ -1,32 +1,29 @@
-A simple web app written in Java using Spring Boot 2.0 that you can use for
-testing. It reads in an env variable `TARGET` and prints "Hello \${TARGET}!". If
-TARGET is not specified, it will use "World" as the TARGET.
+# Hello World - Spring Boot Java
 
-Follow the steps below to create the sample code and then deploy the app to your
-cluster. You can also download a working copy of the sample, by running the
-following commands:
+This guide describes the steps required to create the `helloworld-java-spring` sample app and deploy it to your cluster.
 
-```shell
+The sample app reads a `TARGET` environment variable, and prints `Hello ${TARGET}!`. If `TARGET` is not specified, `World` is used as the default value.
+
+You can also download a working copy of the sample, by running the following commands:
+
+```bash
 git clone -b "{{ branch }}" https://github.com/knative/docs knative-docs
 cd knative-docs/docs/serving/samples/hello-world/helloworld-java-spring
 ```
 
-## Before you begin
+## Prerequisites
 
 - A Kubernetes cluster with Knative installed and DNS configured. Follow the
-  [installation instructions](../../../../install/) if you need to
-  create one.
-- [Docker](https://www.docker.com) installed and running on your local machine,
-  and a Docker Hub account configured (we'll use it for a container registry).
-- You have installed
-  [Java SE 8 or later JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
+  [installation instructions](../../../../install/).
+- [Docker](https://www.docker.com) installed and running on your local machine, and a Docker Hub account configured.
+- (optional) The Knative CLI client [kn](https://github.com/knative/client/releases) can be used to simplify the deployment. Alternatively, you can use `kubectl`, and apply resource files directly.
 
-## Recreating the sample code
+## Building the sample app
 
-1. From the console, create a new empty web project using the curl and unzip
+1. From the console, create a new, empty web project by using the `curl` and `unzip`
    commands:
 
-   ```shell
```

Comment:
I think this isn't indented properly in the preview?
https://deploy-preview-3728--dev-knative.netlify.app/development/serving/samples/hello-world/helloworld-java-spring/#building-the-sample-app

I'd hit that code block and the text under it with an extra tab indent for now til we can clean it up properly

Reply:
same actually for anything under a 1. numbered step because otherwise it's breaking your numbering